guillotina/contrib/cache/strategy.py

```diff
@@ -129,7 +130,9 @@ async def close(self, invalidate=True, publish=True):
                     await self.fill_cache()
                     if len(self._keys_to_publish) > 0 and self._utility._subscriber is not None:
                         keys = self._keys_to_publish
-                        asyncio.ensure_future(self.synchronize(keys))
+                        asyncio.ensure_future(
+                            notice_on_error2("Error on cache synchronize", self.synchronize(keys))
```

Comment:
Why not `notice_on_error`

Reply:
Because it would log `Error on initialize utility {...}`. I guess this message it only makes sense when calling `utility.initialize()`